### DISCUSSION TITLE
index.md not generating links

### DISCUSSION THREAD

**Discussion by thoreinstein** [srid/neuron#638] [General]
I am just getting started trying to convert from that _other_ tool…

I have entered a couple of notes, and linked them.

```
# janders' Digital Garden

This is my digital garden. A place to curate my accumulated knowledge.

- [[Tribes]]
- [[13e4a269]]
```

However the generated page just shows the text as is.

![Screen Shot 2021-07-25 at 7 56 11 PM](https://user-images.githubusercontent.com/719823/126917634-d8f6d9ba-4898-43cf-8c0f-55f7611702ba.png)

If there is a better place to get support or ask this question, please let me know.
Thanks in advance.

**Comment by srid** [comment]
Ah, I see what is happening. You have deactivated the links plugin.

Just delete this line (unless you are specifying all plugins explicitly): https://github.com/janders223/garden/blob/becbaf0e9da0fb0f04f1501e79bbdabe2f507036/neuron.dhall#L7

**Reply by thoreinstein** [reply]
I see, thank you!
